README.md

```diff
@@ -1,7 +1,17 @@
 # opg-sirius-header
-<p> This is a small template repo imported into other Go repos </p>
+<p> 
```

Comment:
My original comment meant more that there is nothing in this readme that would tell someone new to the project how to use this package in another app in order to get the header to show. What to put in the HTML, what to import etc.